client/landing/site.landing/coffee/main.coffee

```diff
@@ -37,7 +37,7 @@ do ->
     new MainController group
 
   kd.config             or= {}
-  kd.config.environment   = if location.hostname is 'koding.com' then 'production' else 'development'
+  kd.config.environment   = window._runtimeOptions.environment
```

Comment:
Only reference to `development` is going to be removed with this pull request. `kd.config.environment` will be compared to `production` only. It sounds good to me.